### PR TITLE
Drop CLA, pure AGPL

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

**Comment by nicpottier** [comment]
@jlovejoy can you give this a look?

**Comment by jlovejoy** [comment]
looks good @nicpottier !  You might want to also add a simply statement on the README along the lines of, "The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license."

(by the way, "AGPL-3.0" is the SPDX short identifier for the license, if you were wondering where that came from: https://spdx.org/licenses/  :)

**Comment by nicpottier** [comment]
Great, thank you!

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,7 @@
 RapidPro - Platform to visually build interactive messaging systems 
 anywhere in the world.
 
-Copyright (C) 2016 Nyaruka, UNICEF
+Copyright (C) 2016 Nyaruka, UNICEF, and individual contributors.
```
Comment (ewheeler) [inline LICENSE]: is it finally time to celebrate 🍾  the new year?

**Pull request merged**